GitTfs/Core/GitTfsRemote.cs

```diff
@@ -695,10 +703,17 @@ private void quickFetch(ITfsChangeset changeset)
         private IEnumerable<ITfsChangeset> FetchChangesets(bool byLots, int lastVersion = -1)
         {
             int lowerBoundChangesetId;
+
+            // If we're starting at the Root side of a branch commit (e.g. C1) but there are invalid commits between C1 and the actual branch-side of the commit operation
```

Comment:
It's actually to help others to be able to quickly review and understand the code without needing to scroll horizontally (which is a huge pain in the butt). While you may have a wide display, not everyone does. In addition, GitHub limits the length of lines shown in PR code reviews, so even if you and I have wide displays, they're virtually useless when reviewing a PR here on GitHub.

Wide monitors are great for watching movies and gaming--not so much for coding.